motr/m0kv/cmd_main.c

```diff
@@ -158,6 +158,12 @@ int main(int argc, char **argv)
 	int           subid;
 	struct params params;
 
+	if (argc == 1)
+        {
+	    usage();
+	    return EXIT_FAILURE;
+        }
```

Comment:
We need to display the usage when the number of arguments are wrong or if any of the arguments/values are invalid.

Reply:
As @huanghua78 pointed out, we already display command usage and help when there are no parameters passed.
@swapnil-seagate, can you take another look and see if the change from the PR is indeed required?

Reply:
@huanghua78,
Earlier ./m0kv used to throw Error 22, as seen above. I added this patch to remove that processing in main and not in library code.

Reply:
With your patch, this `m0kv` will exit 1, and print out help messages.
The only change is the exit code.

I still don't understand why this is needed, or what this patch fixes.

Reply:
If you run ./m0kv, it will print the utility usage, along with ERROR 22
motr[08865]:  20c0  ERROR  [m0kv/cmd_main.c:144:opts_get]  <! rc=-22
Got error -22
Done, rc:  22

This patch will print the same usage details, but without the error 22.
This is the only change in this patch.